pkgs/development/tools/git-repo-updater/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, python37Packages }:
+
+python37Packages.buildPythonApplication rec {
+  pname = "gitup";
+  version = "0.5";
+
+  src = python37Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "11ilz9i2yxrbipyjzpfkj7drx9wkrn3phvd1a60jivphbqdldpgf";
+  };
+
+  propagatedBuildInputs = with python37Packages; [ colorama gitdb2 GitPython smmap2 ];
+
+  meta = with lib; {
+    description = "Easily update multiple Git repositories at once";
+    homepage = https://github.com/earwig/git-repo-updater;
```

Hint: We're probably going to deprecate unquoted URL syntax when RFC 45 is approved.
```suggestion
    homepage = "https://github.com/earwig/git-repo-updater";
```